apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/kontoregisterservice/KontoregisterConsumer.java

```diff
@@ -131,6 +135,19 @@ public Mono<HentKontoResponseDTO> sendHentKontoRequest(String ident) {
                 .flatMap(token -> new SendHentKontoregisterCommand(webClient, requestDto, token.getTokenValue()).call());
     }
 
+    @Timed(name = "providers", tags = {"operation", "kontoregister_slettKonto"})
+    public Mono<List<String>> slettKontoer(List<String> identer) {
+
+        return tokenService.exchange(serviceProperties)
+                .flatMapMany(token -> Flux.range(0, identer.size())
+                        .delayElements(Duration.ofMillis(100))
+                        .map(index -> new SendSlettKontoregisterCommand(
```

Comment:
Er det mulig å gjøre .flatMap() her i stedet for .map() slik at .flatMap(Flux::from) nedenfor blir overflødig?